docs/Jobs/spec.md

```diff
@@ -0,0 +1,63 @@
+# Job spec to  Restfulapi
+
+We use yaml format because it allow comments will note each field if it is required or optional, and gives example whenever possible.
```

Comment:
Change to 
"We use yaml format because it allows commenting each field, e.g. if it is required or optional, and examples whenever possible"